drivers/include/modbus_rtu.h

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (C) 2020-2023 Denis Litvinov <li.denis.iv@gmail.com>
+ * Copyright (C) 2023-2024 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    drivers_modbus_rtu Modbus RTU
+ * @ingroup     drivers
+ * @brief       Modbus RTU
+ *
+ * @{
+ * @file
+ * @brief       Modbus RTU interface definitions
+ *
+ * @author      Denis Litvinov <li.denis.iv@gmail.com>
+ * @author      Bas Stottelaar <bas.stottelaar@gmail.com>
+ */
+
+#ifndef MODBUS_RTU_H
+#define MODBUS_RTU_H
+
+#include "periph/uart.h"
+#include "thread.h"
+#include "mutex.h"
+
+#include "modbus.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Modbus RTU sizes
+ *
+ * @{
+ */
+#define MODBUS_RTU_PACKET_CRC_SIZE              2
+#define MODBUS_RTU_PACKET_REQUEST_SIZE_MIN      8       /**< includes CRC */
+#define MODBUS_RTU_PACKET_RESPONSE_SIZE_MIN     5       /**< includes CRC */
+#define MODBUS_RTU_PACKET_SIZE_MAX              256     /**< includes CRC */
+/** @} */
+
+/**
+ * @brief   Modbus RTU device parameters
+ */
+typedef struct {
+    /**
+     * @brief   RTS pin. @p GPIO_UNDEF if not used.
+     */
+    gpio_t pin_rts;
```

Comment:
why dont you use ?
```suggestion
    gpio_t pin_rts; /**<<  RTS pin. @p GPIO_UNDEF if not used.*/
```

Reply:
I will update this. I just re-used what was already provided in the first PR :-)

Reply:
I have addressed this for `modbus_rtu_params_t` and `modbus_rtu_t`. I did drop the `@internal` annotations, which I could not combine.